test/Psy/Test/TabCompletion/AutoCompleterTest.php

```diff
@@ -21,9 +21,9 @@
 class AutoCompleterTest extends \PHPUnit\Framework\TestCase
 {
     /**
-     * @param $line
-     * @param $mustContain
-     * @param $mustNotContain
+     * @param string $line
+     * @param array  $mustContain
+     * @param array  $mustNotContain
```

Comment:
see, this is why i don't document tests :P

Reply:
also `array` is not very useful here... Just bump to 7.1 and use typehints 😄

Reply:
@bobthecow I don't document they either, but these ones with `dataProvider`s, are important :)

@theofidry Bump to 7.1 is out of the question, I guess :grin: